.github/auto_assign.yml

```diff
@@ -0,0 +1,43 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: false
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 1
+
+# A number of assignees to add to the pull request
+# Set to 0 to add all of the assignees.
+# Uses numberOfReviewers if unset.
+# numberOfAssignees: 2
+
+# Set to true to add reviewers from different groups to pull requests
+useReviewGroups: true
+
+# A list of reviewers, split into different groups, to be added to pull requests (GitHub user name)
+reviewGroups:
+  groupA:
+    - joomlagergerman/reviewer
+
+#  groupB:
+#    - coolwebcreations
+  
+# Set to true to add assignees from different groups to pull requests
+useAssigneeGroups: false
+
+# A list of assignees, split into different froups, to be added to pull requests (GitHub user name)
```

Comment:
Lets remove the connected out lines

Reply:
As a documentation I think that's quite nice and maybe we want to assign it again.

Reply:
hmm would't it be better than to add a link to the docs? ;)

Reply:
done